### ISSUE TITLE
How do users add custom sticker from their phone?

### ISSUE THREAD

**Issue by serdargun** [prscX/react-native-photo-editor#154]
In my app, users need to add their custom stickers like a company  logo or personal logo. So that user will add stickers from their phones. But stickers folder is in app resources folder(ios) or drawable folder(android) and app is encapsulated, users can't reach these folders from their phones. In summary how does users add custom sticker interactively?

**Comment by pramodsharma403** [comment]
@serdargun have you got any solution for that? I have to add stickers from the server URL. is there any way to achieve this task?

**Comment by serdargun** [comment]
Yes but its not from this library. And my solution is a little bit complicated. 

I used following libraries:

1. `react-native-image-picker`
2. `react-native-view-shot`
3. `react-native-fs`

I can pick my background photo and my logo as different components with first library. And I made my logo component draggable. Thus I can put my logo/sticker wherever I want on background image.

To save your new image with logo/sticker, you can use second library. This library doesn't actually save. It just take screenshot of your component.

After that you can save your screenshot photo with using third library.

**Comment by pramodsharma403** [comment]
@serdargun Thanks for the quick response. I will try your suggested solution.

**Comment by Hashir524** [comment]
@serdargun Any code link to your solution?
